zipline/utils/paths.py

```diff
@@ -59,6 +59,21 @@ def ensure_file(path):
     open(path, 'a+').close()  # touch the file
 
 
+def update_modified_time(path, times):
+    """
+    Updates the modified time of an existing file.
```

Comment:
Like `ensure_file`, do you think it would be worth mentioning here that this creates any necessary parent directories?